### PR TITLE
Item Search Api Implementation

### PR DESCRIPTION
**Pull request by kevinle623**
Requesting to merge Search Item Api feature into main branch.

### CODE REVIEW

File: src/main/java/SearchitemsApi.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        }
+        return itemList;
+    }
+
+    private String keywordstext(String keywords, String marketplace) {
+
+        return "https://amazon-price1.p.rapidapi.com/search?keywords="+ keywords.replace(" ", "%20") + "&marketplace=" + marketplace;
+    }
+
+//test
```
Comment (Ljime) [inline src/main/java/SearchitemsApi.java:92]: This comment could be removed

File: src/main/java/Item.java [hunk 2]
```diff
@@ -53,11 +75,37 @@ public void setItemDescription(String description) {
...
         System.out.println("Price: " + itemPrice);
         System.out.println("Date added: " + dateAdded);
+        System.out.println("Last Updated: " + dateLastUpdated);
+        System.out.println("Description:" + itemDescription);
         System.out.println("------------------------------------------");
     }
+
+    public void updatePrice() throws IOException{
+        try {
+            Document doc = Jsoup.connect(url).timeout(10000).userAgent("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36").get();
```
Comment (Ljime) [inline src/main/java/Item.java:96]: Could you explain what this does please? By possibly adding a comment for future reference as I'm not sure what the long string of text does.

**Review by Ljime — CHANGES_REQUESTED** [review]
Overall, looks good, one thing that could change to improve the github work flow is to change the pull request title back to the api call functionality.

Another thing is to add java docs for the functions you created thanks!

File: src/main/java/Item.java (hunk 2 above)
Comment (PyroPoro) [reply src/main/java/Item.java:96]: that line opens a connection to the item's amazon webpage using a specific user agent that bypasses captcha

**Pull request merged**